sdk/python/feast/data_source.py

```diff
@@ -413,9 +413,9 @@ def get_table_query_string(self) -> str:
         raise NotImplementedError
 
 
-class RequestDataSource(DataSource):
+class RequestSource(DataSource):
```

Comment:
I think you'll need to do something like this to deprecate the old class and create the new one :

```
class RequestDataSource(RequestSource):
    def __init__(self):
      warnings.warn(
                  "RequestDataSource is deprecated in favor of RequestSource.",
                  DeprecationWarning,
              )

```

Reply:
Done.